### PR TITLE
Test targeting multiple cross-origin windows.

### PR DESCRIPTION
**Pull request by mikewest**
Addresses https://github.com/whatwg/html/pull/323#commitcomment-14507958.

### CODE REVIEW

**Comment by mikewest** [comment]
@annevk, WDYT?

File: html/browsers/windows/support-open-cross-origin.html [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<meta charset=utf-8>
+<p>Follow this link to open a new browsing context in a separate origin. 
+<a target=second href="http://www2.{{location[host]}}/html/browsers/windows/support-open-cross-origin.html">link</a>.
```
Comment (annevk) [inline html/browsers/windows/support-open-cross-origin.html]: If you want substitution to work you need to have `.sub.html` as filename suffix.

Comment (annevk) [inline html/browsers/windows/support-open-cross-origin.html]: And you probably want `{{domains[www2]}}:{{location[port]}}`.

File: html/browsers/windows/targeting-multiple-cross-origin-manual.html [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<meta charset=utf-8>
+<p>Follow this link to open a new browsing context in a separate origin. Follow the instructions
+in that new window, and then come back to this window.
+<a target=first href="http://天気の良い日.{{location[host]}}/html/browsers/windows/support-open-cross-origin.html">link</a>.
+
+<p>Once you come back to this page, follow this link.
+<a target=second href="support-window-name-echo.html">link</a>.
```
Comment (annevk) [inline html/browsers/windows/targeting-multiple-cross-origin-manual.html]: It seems this cannot work without some other PR landing.

File: html/browsers/windows/targeting-multiple-cross-origin-manual.html [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<meta charset=utf-8>
+<p>Follow this link to open a new browsing context in a separate origin. Follow the instructions
+in that new window, and then come back to this window.
+<a target=first href="http://天気の良い日.{{location[host]}}/html/browsers/windows/support-open-cross-origin.html">link</a>.
```
Comment (annevk) [inline html/browsers/windows/targeting-multiple-cross-origin-manual.html]: Same comment here as above.

File: html/browsers/windows/targeting-multiple-cross-origin-manual.html (hunk 2 above)
Comment (mikewest) [inline html/browsers/windows/targeting-multiple-cross-origin-manual.html]: Yeah. I based this on one of your PRs. :)

File: html/browsers/windows/targeting-multiple-cross-origin-manual.html (hunk 3 above)
Comment (mikewest) [inline html/browsers/windows/targeting-multiple-cross-origin-manual.html]: That explains so much! :)

**Comment by annevk** [comment]
It seems like #2354 should be merged first.

**Comment by gsnedders** [comment]
```
15:01 < aryehgregor> Um, I might have done something wrong by mistake.
15:01 < aryehgregor> I think I pushed to the wrong repo?
```

**Comment by annevk** [comment]
@mikewest it seems if you rebase this we could maybe land this as it seems the PR upon which this depended has landed.

**Comment by mikewest** [comment]
I'll take a look, thanks for the ping.

**Comment by mikewest** [comment]
Updated. Works locally in Chrome. WDYT?

**Comment by annevk** [comment]
Firefox too.

**Pull request merged**